modules/solid_mechanics/src/materials/crystal_plasticity/ComputeElasticityTensorCP.C

```diff
@@ -31,7 +33,9 @@ ComputeElasticityTensorCP::ComputeElasticityTensorCP(const InputParameters & par
                                : nullptr),
     _Euler_angles_mat_prop(declareProperty<RealVectorValue>("Euler_angles")),
     _crysrot(declareProperty<RankTwoTensor>(_base_name + "crysrot")),
-    _R(_Euler_angles)
+    _R(_Euler_angles),
+    _n_euler_angle_vars(
+        isCoupled("euler_angle_variables") ? coupledComponents("euler_angle_variables") : 0)
```

Comment:
is coupled components not simply 0 if isCoupled is false?

Reply:
That's right! - addressed.